etl_tests/test_looker.py

```diff
@@ -0,0 +1,41 @@
+import pytest
```

Comment:
@Iinh could you review this part? Ideally we would have more tests here, but I thought it would be better to do that as a followup (since this is already a fairly large change).

Reply:
this part looks good to me!